R/downsample_for_metrics.R

```diff
@@ -0,0 +1,28 @@
+#' Downsample PCs for use in integration metric calculations 
+#'  
+#' @param integrated_pcs The full set of integrated pcs
```

Comment:
```suggestion
#' @param pcs The PCs to downsample
```
These can be any type of PCs and don't have to be integrated.